lib/generators/doorkeeper/templates/migration.rb

```diff
@@ -41,6 +41,7 @@ def change
       t.datetime :revoked_at
       t.datetime :created_at,        null: false
       t.string   :scopes
+      t.string   :previous_refresh_token
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.